graphql/schema.graphql

```diff
@@ -5284,7 +5284,7 @@ type PullRequestReview implements Node & Comment & Deletable & Updatable & Updat
   """The body of this review rendered as plain text."""
   bodyText: String!
 
-  """A list of review comments for the current pull request review."""
+  """A list of review comments for the checked out pull request review."""
```

Comment:
This is a generated file, so we shouldn't edit it directly 🙈

Reply:
ok...